Makefile

```diff
@@ -1,7 +1,9 @@
 test:
```

Comment:
Like I said, probably best to just store the cert and key in the repo, we don't need to generate them everytime